client/src/components/map/layers/networks-markers/index.tsx

```diff
@@ -97,9 +107,24 @@ const MarkerComponent = ({
   </Marker>
 );
 
-const NetworksMarkers = () => {
+const NetworkMarkersWithData = ({
+  features,
+  isFetched,
+  isError,
+  type,
+  id,
+}: {
+  features: PointFeatureWithNetworkProperties[];
+  isFetched: boolean;
+  isError: boolean;
+  type?: 'organization' | 'project';
+  id?: number;
```

Comment:
```suggestion
  id?: string;
```

Reply:
🤔 That is a number, I transform it in line 246

Reply:
In that case, if it's already a number, you don't have to convert it 5 lines below.